src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -75,8 +75,8 @@ function UpdateService({
         },
         onError: (error: Error) => {
             files.current[0].status = 'error';
-            if (error instanceof ApiError && error.body && isErrorResponse(error.body)) {
-                const response: ErrorResponse = error.body;
+            if (error instanceof ApiError && error.body && isErrorResponse(error)) {
```

Comment:
the isErrorRepsonse method is already checking `error instanceof ApiError && error.body`. So remove this from all other calling methods.

Reply:
fixed.